tests/utils/visit-block-page.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import {
+	createNewPost,
+	visitAdminPage,
+	insertBlock,
+	getEditedPostContent,
+} from '@wordpress/e2e-test-utils';
+import { outputFile } from 'fs-extra';
+import { dirname } from 'path';
+import kebabCase from 'lodash/kebabCase';
+
+async function visitPage( link ) {
+	await page.goto( link );
+	const isWelcomeGuideActive = await page.evaluate( () =>
+		wp.data.select( 'core/edit-post' ).isFeatureActive( 'welcomeGuide' )
+	);
+	const isFullscreenMode = await page.evaluate( () =>
+		wp.data.select( 'core/edit-post' ).isFeatureActive( 'fullscreenMode' )
+	);
+
+	if ( isWelcomeGuideActive ) {
+		await page.evaluate( () =>
+			wp.data.dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' )
+		);
+	}
+
+	if ( isFullscreenMode ) {
+		await page.evaluate( () =>
+			wp.data
+				.dispatch( 'core/edit-post' )
+				.toggleFeature( 'fullscreenMode' )
+		);
+	}
+}
```

Comment:
discovering I need this took me a lot of time, you can’t click the inserter without toggling off the welcome guide, same for fullscreenMode.
When creating a new post, this is done directly in the `createNewPost` helper, since we navigate to this post, we need to do it.
I didn’t test without fullscreen, so I might try deleting it and seeing if the test still pass.

Reply:
Fullscreen was not needed, deleted in 9d6ee8e